File: src/FrontEnd/Client/src/app/_models/organization.ts
export interface Organization {
    id: string;
    published: Date;
    updated: Date;
    level: OrganizationLevel;
    region: OrganizationRegion;
    address: Address;
    isActive: boolean;
    organizationName: OrganizationName;
    specializationIds: string[];
}

export enum OrganizationLevel {
    AreaLevel,
    RegionLevel,
    RepublicLevel
}

export interface OrganizationName {
    usualName: string;
    officialName: string;
}

export enum OrganizationRegion {
    BrestRegion = 'Брестская область',
    VitebskRegion = '',
    GomelRegion = '',
    GrodnoRegion = '',
    MinskRegion = '',
    MogilevRegion = '',
    Minsk = '',
    Belarus = "Республика Беларусь"
}

export interface Address {
    line: string;
}

File: src/Services/Organizations/GRPC/Organization.GRPC/Entities/Organization.cs
using Organization.GRPC.Entities.Common;

namespace Organization.GRPC.Entities
{
    public class Organization : BaseDomainEntity
    {
        public OrganizationLevel Level { get; set; }
        public OrganizationRegion Region { get; set; }
        public Address Address { get; set; }
        public bool IsActive { get; set; }
        public OrganizationName OrganizationName { get; set; }
        public List<string> SpecializationIds { get; set; }
    }
}
